CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # Changelog of the Jest Coverage Comment
 
+## [Jest Coverage Comment 1.0.35](https://github.com/MishaKav/jest-coverage-comment/tree/v1.0.35)
+
+**Release Date:** 2026-08-01
+
+#### Changes
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)